### PR TITLE
Renames & removes

### PR DESCRIPTION
**Pull request by Denperidge**
Note; the following commits are suggestions for a new structure. Please feel free to disagree and discuss! 
- Agenda-Items' routes, controllers and templates now follow the same structure sessions
- Removed unused *.ts & *.hbs files

### CODE REVIEW

**Review by aliokan — APPROVED** [review]
I've wanted to do it for a while, thank you! :pray:

File: app/components/participant-card.hbs
Comment (Windvis) [inline app/components/participant-card.hbs:1]: This component doesn't seem to be in-use anymore? Maybe we can just fully remove it?

**Review by Windvis — COMMENTED** [review]
Other than the minor remark, this looks good to me 👍

**Review by aliokan — APPROVED** [review]
LGTM

**Pull request merged**